architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,12 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, projects hibernate after 30 minutes of inactivity.
```

Comment:
@ahardin-rh @sallyom There is a idle_threshold actually, if the the network activity received is less than the idle_threshold the project will be idled. I think this might need to be referred to in the doc.